### PR TITLE
new documentation for aspiring greenbox developers & new readme

### PR DESCRIPTION
**Pull request by docrostov**
pretty simple PR. only touches two files -- it updates the readme to be more descriptive and include the install/run procedure, and it adds a new much longer file within a /documentation/ folder that serves broadly as a PR walkthrough, explaining the three sections of the code, things that each does, and the various ones that a specific PR might want to touch (with linked examples for most of them). also includes your messages from the discord on how to set up a local copy tho i rather ironically haven't tested it yet

### CODE REVIEW

**Comment by gausie** [comment]
I think it might be good to split this up into multiple README.mds - we can have one per package folder and one in the root packages folder explaining the monorepo.

**Comment by docrostov** [comment]
sounds good. will do that tonight

**Review by Tokoeka — DISMISSED** [review]

File: packages/greenbox-web/README.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+The same is true of many of the items within components, but not all. For instance, `AlphaImage.tsx` is a helper function built by Gausie that essentially builds transparency where none exists for the black-and-white images KOL is known for, allowing for polyfill of the whitespace when you are filling a box green for the web interface. `Image.tsx` simply redirects images to KOL's image server, allowing us to glide on the coattails of TPTB's web hosting for the six people in the known universe who use our script. The components have a lot of different elemnts that are bespoke and relevant only to that specific component. Please dig through these and attempt to understand their working patterns before modifying them.
+
+Within `greenbox-web/src/store`, you will find the .ts files that are used to combine all the myriad components into a final website. The primary file here is (fittingly) `index.ts`, which imports essentially everything from `components` and plops them all down into a file that users can view.
```
Comment (gausie) [inline packages/greenbox-web/README.md:20]: This isn't quite right - the store folder manages the sort of shared brain of all of the components - our application state

Comment (gausie) [reply packages/greenbox-web/README.md:20]: The enploppening happens in main.tsx

**Comment by gausie** [comment]
Super great. I would like to subscribe to your newsletter and also hire you to document code

**Pull request merged**